### PR TITLE
Added a model card README.md for my pretrained model.

### PR DESCRIPTION
**Pull request by Pradhy729**

### CODE REVIEW

File: model_cards/pradhyra/AWSBlogBert/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This model is pre-trained on blog articles from AWS Blogs. 
+Training code can be found at this [url]()
```
Comment (JetRunner) [inline model_cards/pradhyra/AWSBlogBert/README.md:2]: Do you forget to add the URL?

Comment (Pradhy729) [reply model_cards/pradhyra/AWSBlogBert/README.md]: Sorry - I actually meant to delete that line. It is in a private repo that I'm not able to link to.

**Comment by julien-c** [comment]
Nice work. What's your intended use case, @Pradhy729? Topic classification? Something else?

**Comment by Pradhy729** [comment]
Thanks! A few different use cases actually. Better entity recognition, topic classification, sentiment extraction and hopefully play around with summarization as well.

**Pull request merged**